exercises/practice/dnd-character/.meta/example.rb

```diff
@@ -0,0 +1,23 @@
+class DndCharacter
+  attr_reader :strength, :dexterity, :constitution, :intelligence, :wisdom, :charisma, :hitpoints
+
+  def initialize
+    @strength = roll_dice
+    @dexterity = roll_dice
+    @constitution = roll_dice
+    @intelligence = roll_dice
+    @wisdom = roll_dice
+    @charisma = roll_dice
+
+    @hitpoints = 10 + self.class.modifier(constitution)
```

Comment:
This is known from the constitution and we can derive the adjustment from base 10 hitpoints.

We might want to test through the story of the character and not the story of "modifier" as a function in a language.